doc/how_to/notebook/other_nb.md

```diff
@@ -8,6 +8,12 @@ This guide addresses how to develop apps in Google Colab, VSCode, nteract, and o
 
 In the Google Colaboratory notebook, first make sure to load the `pn.extension()`. Panel objects will then render themselves if they are the last item in a notebook cell. Please note that in Colab rendering for each notebook cell is isolated, which means that every cell must reload the Panel extension code separately. This will result in somewhat slower and larger notebook than with other notebook technologies.
 
+* Be aware: using Panel with ipywidgets in Colab does not work, due to Colab's architecture constraints.
+
+### Anaconda Cloud
+
+When using Anaconda Coud, make sure to upgrade the Panel package to the latest version using !pip install
```

Comment:
```suggestion
When using Anaconda Cloud, make sure to upgrade the Panel package to the latest version using !pip install
```

Reply:
Suggested addition below new text.

Best practice is to always

-Check for the latest version of Panel (with command ...)
-If not on the latest version, upgrade to Panel to the latest version with !pip install. See installation guide for further details.